test/is_palindrome.hpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+
+bool isPalindrome(int x) {
```

Comment:
https://google.github.io/styleguide/cppguide.html#Function_Names

```suggestion
bool IsPalindrome(int x) {
```